ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -166,6 +162,15 @@ def pytest_configure(config):
                 config._metadata['OCS operator'] = (
                     ocs_catalog.get_image_name()
                 )
+            mods = get_version_info(
+                namespace=ocsci_config.ENV_DATA['cluster_namespace']
+            )
+            skip_list = ['ocs-operator']
+            for key, val in mods.items():
+                if key in skip_list:
+                    pass
+                else:
+                    config._metadata[key] = val.rsplit('/')[-1]
```

Comment:
Can be shorter with a negative condition without the else clause:
 
if not key in skip_list:
    config._metadata[key] = val.rsplit('/')[-1]

Reply:
Done